### PR TITLE
#9721 - Remove `tui-table` classes from demo

### LINKED ISSUE #9721: 🐞 - Demo should not use any of "tui-" classes other than spacings
### Reproduction url

https://taiga-ui.dev/form

### Description

Some of the core layouting building blocks used in [Forms](https://taiga-ui.dev/form) is missing documentation. These elements (like row, skeleton, table) seem to be useful for a lot of other contexts, but some more documentation how they are intended to use them would be helpful.

### Taiga UI version

4.14.0

### Browser Used

Firefox

### OS Used

Windows

### PR DESCRIPTION
**Pull request by shiv9604**
Fixes #9721 

**Changes Description:-**

Migrated tui-form classes from the below pages.
1. [Dropdown Context > Context Menu Example](https://taiga-ui.dev/directives/dropdown-context)
2. [Browser Support](https://taiga-ui.dev/browser-support)
3. [Filter Pipe Usage](https://taiga-ui.dev/pipes/filter)

### CODE REVIEW

File: projects/demo/src/modules/directives/dropdown-context/examples/2/index.ts [hunk 1]
```diff
@@ -2,12 +2,34 @@ import {NgForOf} from '@angular/common';
 import {Component, inject} from '@angular/core';
 import {changeDetection} from '@demo/emulate/change-detection';
 import {encapsulation} from '@demo/emulate/encapsulation';
+import {
+    TuiTable,
```
Comment (waterplea) [inline projects/demo/src/modules/directives/dropdown-context/examples/2/index.ts:6]: `TuiTable` is enough, it's already a constant with all other standalone directives:
https://github.com/taiga-family/taiga-ui/blob/main/projects/addon-table/components/table/table.ts

File: projects/demo/src/modules/directives/dropdown-context/examples/2/index.less [hunk 2]
```diff
@@ -1,7 +1,23 @@
...
+    inline-size: 100%;
+}
+
+[tuiTh],
+[tuiTd] {
+    border-inline-start: none;
+    border-inline-end: none;
+}
+
+[tuiTd] {
```
Comment (waterplea) [inline projects/demo/src/modules/directives/dropdown-context/examples/2/index.less:21]: Let's also remove `border-block-start` form `[tuiTh]` so it looks more like it used to.

File: projects/demo/src/modules/info/browsers/index.less [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    border-inline-start: none;
+    border-inline-end: none;
+}
+
+[tuiTd] {
+    font: var(--tui-font-text-m) !important;
+}
+
+td {
+    inline-size: 18.75rem;
```
Comment (waterplea) [inline projects/demo/src/modules/info/browsers/index.less:18]: Why is it not under `[tuiTd]` above?

File: projects/demo/src/modules/info/browsers/index.ts [hunk 4]
```diff
@@ -1,11 +1,29 @@
...
+    TuiTableThGroup,
+    TuiTableTr,
+} from '@taiga-ui/addon-table';
 
 @Component({
     standalone: true,
-    imports: [TuiDemo],
+    imports: [
+        TuiDemo,
+        TuiTable,
```
Comment (waterplea) [inline projects/demo/src/modules/info/browsers/index.ts:17]: Same comment

File: projects/demo/src/modules/pipes/filter/examples/1/index.ts [hunk 5]
```diff
@@ -2,6 +2,14 @@ import {NgForOf} from '@angular/common';
 import {Component} from '@angular/core';
 import {changeDetection} from '@demo/emulate/change-detection';
 import {encapsulation} from '@demo/emulate/encapsulation';
+import {
+    TuiTable,
```
Comment (waterplea) [inline projects/demo/src/modules/pipes/filter/examples/1/index.ts:6]: And here

**Review by waterplea — APPROVED** [review]
Thank you! A few minor comments

File: projects/demo/src/modules/info/browsers/index.less (hunk 3 above)
Comment (shiv9604) [reply projects/demo/src/modules/info/browsers/index.less:18]: It was existing code so I didn't touch that, Let me transform to attribute selector under `[tuiTd]`.

File: projects/demo/src/modules/directives/dropdown-context/examples/2/index.ts (hunk 1 above)
Comment (shiv9604) [reply projects/demo/src/modules/directives/dropdown-context/examples/2/index.ts]: Ahh, I see! Thanks for informing me.

Resolved!

File: projects/demo/src/modules/directives/dropdown-context/examples/2/index.less (hunk 2 above)
Comment (shiv9604) [reply projects/demo/src/modules/directives/dropdown-context/examples/2/index.less:25]: Yeah, header border is appearing

Comment (shiv9604) [reply projects/demo/src/modules/directives/dropdown-context/examples/2/index.less:25]: Resolved!

File: projects/demo/src/modules/directives/dropdown-context/examples/2/index.ts (hunk 1 above)
Comment (shiv9604) [reply projects/demo/src/modules/directives/dropdown-context/examples/2/index.ts]: Resolved!

File: projects/demo/src/modules/info/browsers/index.less (hunk 3 above)
Comment (shiv9604) [reply projects/demo/src/modules/info/browsers/index.less:19]: Resolved!

File: projects/demo/src/modules/info/browsers/index.ts (hunk 4 above)
Comment (shiv9604) [reply projects/demo/src/modules/info/browsers/index.ts]: Resolved!

**Review by splincode — APPROVED** [review]

**Pull request merged**